gcode_export.py

```diff
@@ -296,7 +297,7 @@ def execute(self, context):
                 folder = '//' + os.path.splitext(bpy.path.basename(bpy.context.blend_data.filepath))[0]
             else:
                 folder = props.folder
-            if '.gcode' not in folder: folder += '.gcode'
+            #if '.gcode' not in folder: folder += '.gcode'
```

Comment:
Hi @luigipacheco I assume that you want to use a different format with Kuka. Could you use a condition for such case? So that for regular gcode files, it still adds the extension.

Reply:
Yes ill review this, i have a version with the kuka post processor working but it requires some setup and im not sure its safe for someone to use without understanding it, but i can clean the gcode exporter to match this.